web-animations-1/Overview.bs

```diff
@@ -3188,9 +3189,14 @@ Keyframe effects {#keyframe-effects}
 ----------------------------------------
 
 <dfn lt="keyframe effect">Keyframe effects</dfn> are a kind of <a>animation
-effect</a> that use the output of the timing model to update CSS properties for
-an element or pseudo-element such as <code>::before</code> or
-<code>::after</code> [[!SELECT]] referred to as the <dfn>target element</dfn>.
+effect</a> that use the output of the timing model to update CSS properties of
+an element known as the <dfn>target element</dfn> or one of its <a>pseudo-elements</a>
+(in which case the target element is the <a>originating element</a> of the pseudo-element).
+
+The combination of <a>target element</a> and <a>pseudo-element</a> selector
+referred to as the <dfn>effect target</dfn>. In the case that the effect target does not exist
+(for example when it targets a `::after` pseudo-element where one does not exist)
+the effect applies no styles until the target comes into existence.
```

Comment:
I'd rather we don't change the meaning of "target element". This should be an API-only change, the model itself should stay the same, right?

Reply:
Actually, may it's ok. I'd probably just prefer to reword this so that the definition of "effect target" comes before "target element".

So the first paragraph should be just:

> Keyframe effects are a kind of animation effect that use the output of the timing model to update CSS properties for
an element or pseudo-element such as ::before or ::after [[!SELECT]] referred to as the effect target.

Reply:
Good idea to reorder.